packages/server/src/insights/dashboard.entity.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  BaseEntity,
+  Column,
+  Entity,
+  ManyToOne,
+  PrimaryColumn,
+  Unique,
+} from 'typeorm';
+import { UserCourseModel } from '../profile/user-course.entity';
+import { InsightDetail } from '@koh/common';
+
+@Entity('insight_dashboard_model')
+@Unique('dashboardKey', ['name', 'userCourseId'])
+export class InsightDashboardModel extends BaseEntity {
+  // CourseSettings existence is dependent on the existence of a course
```

Comment:
```suggestion
  // existence is dependent on the existence of a course
```

Reply:
lol this is from me copying the entity definition for course settings, I'm gonna remove the comment entirely